docs/dev/table/connectors/formats/canal.md

```diff
@@ -292,14 +292,14 @@ Format Options
       <td>optional</td>
       <td style="word-wrap: break-word;">(none)</td>
       <td>String</td>
-      <td>Only read changelog rows which match the specific database (by comparing the "database" meta field in the Canal record).</td>
+      <td>Only read changelog rows which match the specific database (by regular matching the "database" meta field in the Canal record).</td>
```

Comment:
What do you think about improving the description to the following?

```
An optional regular expression to only read the speicifc databases changelog rows by regular matching the "database" meta field in the Canal record. The pattern string is compatible with Java's <a href="https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html">Pattern</a>.
```

Would be great if you can update the description in `org.apache.flink.formats.json.canal.CanalJsonOptions#DATABASE_INCLUDE`.